Code/Tools/AssetProcessor/native/tests/AssetProcessorMessagesTests.cpp

```diff
@@ -123,7 +123,9 @@ namespace AssetProcessorMessagesTests
             ASSERT_EQ(status, ApplicationManager::BeforeRunStatus::Status_Success);
 
             m_batchApplicationManager->m_platformConfiguration = new PlatformConfiguration();
-            m_batchApplicationManager->InitAssetProcessorManager();
+            
+            AZStd::vector<ApplicationManagerBase::APCommandLineSwitch> commandLineInfo;
```

Comment:
should the test inspect some/any of the switches? 
or at least not empty?

Reply:
I don't like how the command line flags are currently setup for Asset Processor, so I think we should do a deeper dive on improving this system, which should include some expanding tests. I've made an internal ticket to track that.